app/src/main/java/com/worldsnas/starplayer/view/musics_list/MusicsListFragment.kt

```diff
@@ -32,14 +39,11 @@ class MusicsListFragment : Fragment() {
 
         daggerSetup()
 
-        val musics = ArrayList<Music>()
-        musics.add(Music(0, "Test", "Ali", "Single", "Pop","somewhere"))
-        musics.add(Music(0, "Test", "Ali", "Single", "Pop","somewhere"))
-        musics.add(Music(0, "Test", "Ali", "Single", "Pop","somewhere"))
-        musics.add(Music(0, "Test", "Ali", "Single", "Pop","somewhere"))
-        musics.add(Music(0, "Test", "Ali", "Single", "Pop","somewhere"))
-        musics.add(Music(0, "Test", "Ali", "Single", "Pop","somewhere"))
-        binding.recyclerview.adapter = MusicsListAdapter(musics)
+        val musicObserver = Observer<List<Music>> {
+
+            binding.recyclerview.adapter = MusicsListAdapter(it as ArrayList<Music>)
```

Comment:
in kare dorosti nist
shoma bayad data dakhele adapter ro beroz konid na inke khode adapter ro avaz konid

Reply:
yani adapter ro az Observer biaram birun?
ya faghat val music = it bezaram

Reply:
adapter hatman bayad biad biron